core/src/integration-test/java/com/scalar/db/storage/dynamo/DynamoMultipleClusteringKeysIntegrationTest.java

```diff
@@ -0,0 +1,698 @@
+package com.scalar.db.storage.dynamo;
```

Comment:
Similar to `IntegrationTestBase`, I think we can run this nice integration test for all the adapters. Can you please do that in separate PR first?

Reply:
Thank you. I'll do it